src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,57 @@
+package core.basesyntax;
+
+import core.basesyntax.converter.DataConverter;
+import core.basesyntax.converter.DataConverterImpl;
+import core.basesyntax.db.FruitDao;
+import core.basesyntax.db.FruitDaoImpl;
+import core.basesyntax.file.operations.FIleWriterImpl;
```

Comment:
The class name `FIleWriterImpl` seems to be incorrectly capitalized. It should be `FileWriterImpl` to match Java naming conventions and ensure correct class reference.